src/client/components/AddResume/AddResume.js

```diff
@@ -1,48 +1,105 @@
-import React, {useState} from 'react';
+import React, { useState } from 'react';
 import ReactDom from 'react-dom';
 import Button from '../Button/Button';
 import './AddResume.css';
 import close from '../../assets/images/closeIcon.svg';
+import { Link } from 'react-router-dom';
 
-export const AddResume = () => {
-const[title,setTitle] = useState('');
-const[description,setDescription] = useState('');
-const[uploadFile,setUploadaFile] =useState('');
+export const AddResume = ({ uploadedFile, setUploadedFile }) => {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [error, setError] = useState(null);
 
-    return ReactDom.createPortal ((
-        <div className = "popup">
-            <div className = "popup-form">
-                <img className="close-icon" src= {close} alt='close Icon'/>
-                <h3>Upload new CV</h3>
-            <div className='form'>
-                <div className = "title">
-                    <label>Add Title</label>
-                    <div>
-                    <input type='text' value={title} onChange={(e)=> setTitle(e.target.value)}/>
-                    </div>
-                </div>
-                <div className = "description">
-                    <label>Description</label>
-                    <div>
-                    <textarea type='text' value={description} onChange={(e)=> setDescription(e.target.value)}/>
-                    </div>
-                </div>
-                <div className = "upload">
-                    <label>Select a file 
-                    <span className = "support-text"> * File supported DOC,DOCX,PDF,RTF,TXT, 5MB Max</span>
-                    </label>
-                    <div>
-                    <input type='text' value={uploadFile} onChange={(e)=> setUploadaFile(e.target.value)}/>
-                    <Button buttonName='Browse'/>
-                    </div>
-                </div>
+  const types = [
+    'application/pdf',
+    'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+  ];
+
+  const handleChange = (e) => {
+    const selectedFile = e.target.files[0];
+    setUploadedFile(selectedFile);
+
+    if (selectedFile) {
+      if (types.includes(selectedFile.type)) {
+        setError(null);
+        setUploadedFile(selectedFile);
+      } else {
+        setUploadedFile(null);
+        setError('Please select a pdf file');
+      }
+    }
+  };
+
+  return ReactDom.createPortal(
+    <div className="popup">
+      <div className="popup-form">
+        <img className="close-icon" src={close} alt="close Icon" />
```

Comment:
this close icon should have some functionality that pop-up should be closed when we click on that 
```suggestion
const [close, setClose] = useState(false);
        <img className="close-icon" src={close} alt="close Icon" onClick={()=>setClose(false)} />
```

Reply:
I think it is not mentioned in my story. @dpfernandes please advice.